clients/python/test/e2e/test_files_api.py

```diff
@@ -90,9 +89,7 @@ def test_search_files(
     use_id: bool,
     faker: Faker,
 ) -> None:
-    results: PaginationIterable = files_api._search_files(
-        sha256_checksum=f"{faker.sha256()}"
-    )
+    results = files_api._search_files(sha256_checksum=f"{faker.sha256()}")
```

Comment:
aha ... tests are not backwareds compatible?!

Reply:
yes. This is always the pain with these tests.